x-pack/plugins/apm/public/components/app/service_inventory/index.tsx

```diff
@@ -82,14 +132,30 @@ function useServicesFetcher() {
         ),
       });
     }
-  }, [data.hasLegacyData, upgradeAssistantHref, core.notifications.toasts]);
+  }, [
+    mainStatistics.hasLegacyData,
+    upgradeAssistantHref,
+    core.notifications.toasts,
+  ]);
 
-  return { servicesData: data, servicesStatus: status };
+  return {
+    servicesData: mainStatistics,
+    servicesStatus: status,
```

Comment:
Are we only interested in the `mainStatistics` status, and not also the comparison status?

Reply:
Yes, the status from the `mainStatistics` is going to be used to show an error message if that was the case. And if any happened while fetching the comparison statistics it just wouldn't show the sparklines.

Reply:
But the comparison statistics is used to define if the loading spinner should be visible or not. https://github.com/elastic/kibana/pull/107094/files#diff-1f8752bfc4d79520010b5d507b96176f454830da1dc8e2091578f66e42007d02R145-R148